dataset/dictionary/organizations.json5

```diff
@@ -386,17 +387,22 @@
   {
     en: "The Eremites",
     ja: "エルマイト旅団",
+    zhCN: "镀金旅团",
     pronunciationJa: "エルマイトりょだん",
     tags: [ "sumeru", "organization" ],
   },
   {
     en: "Amurta",
     ja: "生論派",
+    zhCN: "生论派",
     tags: [ "sumeru", "organization" ],
+    // Note: I don't think tag "organization" is suitable for this and the following.
+    // Because they are just academic genres.
```

Comment:
You are right.
Can you remove `organization` tag from Amurta and Haravatat, and put them before Akasha in story.json5?

Reply:
OK